packages/game-passes/contracts/GamePasses.sol

```diff
@@ -672,9 +685,13 @@ contract GamePasses is
             revert InvalidSender();
         }
 
+        if (burnId == mintId) {
+            revert SameTokenIdBurnAndMint(burnId);
+        }
+
         TokenConfig storage burnConfig = _tokenStorage().tokenConfigs[burnId];
         if (!burnConfig.isConfigured) {
-            revert BurnMintNotConfigured(burnId);
+            revert TokenNotConfigured(burnId);
```

Comment:
This check is missing in operatorBatchBurnAndMint ?

Reply:
Good call, added it